packages/eas-cli/src/build/android/syncProjectConfiguration.ts

```diff
@@ -16,20 +16,20 @@ export async function syncProjectConfigurationAsync({
   projectDir,
   exp,
   localAutoIncrement,
-  nonInteractive,
+  projectId,
```

Comment:
if we know that projectId is enforced in all commands, we can just read it from `exp.extra.projectId`, passing it here seems redundant.

the same applies to most changes in this PR

Reply:
The downside of reading from `exp.extra.projectId` is that it is typed as `string | undefined`. But it does seem like these cases of reading the config multiple times and it varying breaks my current assumptions about projectId context, so these will probably need some other type of context or something.